posthog/apps.py

```diff
@@ -32,3 +33,13 @@ def ready(self):
             posthoganalytics.disabled = True
         elif settings.TEST or os.environ.get("OPT_OUT_CAPTURE", False):
             posthoganalytics.disabled = True
+
+        for version_requirement in settings.VERSION_REQUIREMENTS:
+            [in_range, version] = version_requirement.is_service_in_accepted_version()
+            if not in_range:
+                start_anyway = input(
```

Comment:
is this something we would like to support? If yes, we should also add a `--skip-version-requirements` or `--force` flag to skip this check entirely

Reply:
Not in the future, no. But as a first step, yes. I just want to see how this behaves out in the wild before being strict.

From the PR description:

> Why do we not throw if the version is invalid?
> 
> I wanted to start with a soft warning that can be overridden until we try this in production for a bit and introduce hard requirements.


Maybe we could send a PostHog event to get a sense of people being out of range 🤔